### DISCUSSION TITLE
Rag pipeline

### DISCUSSION THREAD

**Discussion by lakshita470** [DrAlzahraniProjects/csusb_fall2024_cse6550_team4#35] [General]
I will conduct session on RAG pipeline, and langchain 
Everyone regardless of documentation and Infrastructure team join the session , everyone should have basic knowledge of workflow.

Confirm day and time in comments. 
Thursday or Friday

**Comment by Fizashaik12** [comment]
Thursday after class

**Comment by venkat-coyote** [comment]
Thursday
